### PR TITLE
style: bignumber spaces

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: #4779 

https://user-images.githubusercontent.com/962095/227583556-fa6b6596-08d5-4214-a986-a6a2cbc274df.mp4

### CODE REVIEW

File: packages/frontend/src/components/SimpleStatistic/SimpleStatistics.styles.tsx [hunk 1]
```diff
@@ -1,13 +1,25 @@
 import { Colors } from '@blueprintjs/core';
 import styled from 'styled-components';
 
-export const BigNumberContainer = styled.div`
+interface BigNumberContainerProps {
+    $paddingBottom?: number;
```
Comment (yasmin-e) [inline packages/frontend/src/components/SimpleStatistic/SimpleStatistics.styles.tsx:5]: just a small comment: why is it $paddingBottom and not just paddingBottom? 😅

**Review by yasmin-e — APPROVED** [review]

File: packages/frontend/src/components/SimpleStatistic/SimpleStatistics.styles.tsx (hunk 1 above)
Comment (IrakliJani) [reply packages/frontend/src/components/SimpleStatistic/SimpleStatistics.styles.tsx:5]: @yasmin-e  if you prefix with dollar sign then the argument will not be passed down to the dom node. see this for
more: https://styled-components.com/docs/api#transient-props

**Comment by PriPatel** [comment]
awesome

**Pull request merged**